docs/integrations/integrations_overview.md

```diff
@@ -14,7 +14,7 @@ The current Continuous Integration (CI) platforms/environments supported are:
 
 | Platform/Environment | Information Link |
 | -------------------- | ---------------- |
-| GitHub App | [GitHub Marketplace][github_app_docs] |
+| GitHub App | [Documentation][github_app_docs] |
```

Comment:
I actually think we should have both the doc and marketplace links. You can separate with a semicolon or whatever looks best to differentiate.